lightning/src/ln/channelmanager.rs

```diff
@@ -13376,12 +13129,14 @@ where
 	/// [`InvoiceRequest::quantity`]: crate::offers::invoice_request::InvoiceRequest::quantity
 	pub fn pay_for_offer_with_quantity(
 		&self, offer: &Offer, amount_msats: Option<u64>, payment_id: PaymentId,
-		optional_params: OptionalOfferPaymentParams, quantity: u64,
+		custom_tlvs: Vec<(u64, Vec<u8>)>, optional_params: OptionalOfferPaymentParams,
```

Comment:
These should really go in `OptionalOfferPaymentParams`, given that exists to hide fields we expect users to never set.